CHANGELOG.md

```diff
@@ -70,6 +70,7 @@ This means that druid no longer requires cairo on macOS and uses Core Graphics i
 - `SHOW_WINDOW` and `CLOSE_WINDOW` commands now only use `Target` to determine the affected window. ([#928] by [@finnerale])
 - Replaced `NEW_WINDOW`, `SET_MENU` and `SHOW_CONTEXT_MENU` commands with methods on `EventCtx` and `DelegateCtx`. ([#931] by [@finnerale])
 - Replaced `Command::one_shot` and `::take_object` with a `SingleUse` payload wrapper type. ([#959] by [@finnerale])
+- Renamed `BaseState` to `WidgetState` ([#969] by [@cmyr])
```

Comment:
`BaseState` / `WidgetState` are not public API right? In that case this is not really a change for users.
I believe it should go into the `Maintenance` section instead.

Reply:
cool, I think this should be ready to go now, pending CI.